.github/workflows/_deploy_kafka_test.yml

```diff
@@ -0,0 +1,198 @@
+name: Setup Workload Identity 
+
+on:
+  workflow_call:
+    inputs:
+      skr-client-akv-endpoint:
+        description: "AKV endpoint for key release"
```

Comment:
Since this is mostly internal use, I would set some of these inputs as Github Secrets for the repo.

Reply:
I'm thinking that we can set AKV endpoint as a variable instead of a secret. Since not everyone has the access to the settings tab to our repo, I would think setting akv endpoint as a variable should be sufficient?

Reply:
The idea of it being a secret is for security, which means it would be a good thing that people don't have access to the settings tab. I'm not sure of problems putting the AKV/MAA endpoint names in a public repo could cause, but I think it would be a good thing to ask Dominic why he made these secrets in the ACI examples repo.

Reply:
done